Project/Fall2020_CSC403_Project/FrmBattle.cs

```diff
@@ -86,37 +86,48 @@ public partial class FrmBattle : Form {
             {
                 enemy.OnAttack(enemy.Attacks[1].atkVal * -1);
             }
-        }
-        
-      UpdateHealthBars();
-      if (player.Health <= 0 || enemy.Health <= 0) {
-        instance = null;
-        Close();
-      }
-    }
-
-    // placeholder EventHandler function for the items button
-    private void btnItems_Click(object sender, EventArgs e)
+        }
+        
+      UpdateHealthBars();
+      if (player.Health <= 0 || enemy.Health <= 0) {
+        instance = null;
+        Close();
+      }
+
+      // adding random dialogue to the Battle using the textboxes
+      Random random1 = new Random();
+      Random random2 = new Random();

```

Comment:
You shouldn't need multiple random objects to generate random numbers

Reply:
You're right. I'll just use random1. Thank you.